nixos/modules/services/web-apps/filesender.nix

```
@@ -0,0 +1,403 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  cfg = config.services.filesender;
+  fpm = config.services.phpfpm.pools.filesender;
+
+  ssp = cfg.simpleSamlPhp.package;
+
+  phpPackage = pkgs.php.withExtensions (
+    { enabled
+    , all
+    }: with all;
+    [
+      xml
+      mbstring
+    ]
+    ++ enabled
+  );
+
+  phpFile = x: "<?php\n" + x;
```

Why no close bracket at the end?
```suggestion
  phpFile = x: "<?php\n" + x + "\n>";
```

---

According to [PHP docs](https://www.php.net/manual/en/language.basic-syntax.phptags.php), "If a file contains only PHP code, it is preferable to omit the PHP closing tag at the end of the file."